### PR TITLE
[Beta] Sandpack: upgrade CodeMirror

### PR DESCRIPTION
**Pull request by danilowoz**
This updates Sandpack to the latest version, which contains the CodeMirror upgrade and addresses some issues on the syntax highlight (mentioned on https://github.com/reactjs/reactjs.org/pull/5368). Plus, it makes us up to date with the latest version of CodeMirror, which makes it easier to pull down any future improvements and bug fixes.

However, I think there are a few things we'll have to afford in order to get this merged @gaearon: the latest CodeMirror version increases the bundle size a bit (~18kb), but I guess it's fine as long as we lazily load Sandpack dependencies.

### CODE REVIEW

File: beta/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "check-all": "npm-run-all prettier lint:fix tsc"
   },
   "dependencies": {
-    "@codesandbox/sandpack-react": "1.15.5",
+    "@codesandbox/sandpack-react": "https://pkg.csb.dev/codesandbox/sandpack/commit/e2e4cd70/@codesandbox/sandpack-react",
```
Comment (danilowoz) [inline beta/package.json:25]: TODO: get https://github.com/codesandbox/sandpack/pull/672 merge first

**Comment by gaearon** [comment]
Does this include https://github.com/codesandbox/sandpack/pull/672?

**Comment by danilowoz** [comment]
Yes, it does. But for some reason, the syntax highlight didn't work correctly on the playground editor, but it did on all the other cases. Any idea why?

**Comment by danilowoz** [comment]
@gaearon I'm wondering if there's any blocker to get this merged

**Pull request closed without merging**